src/Microsoft.Restier.Core/ApiBase.cs

```diff
@@ -67,7 +67,14 @@ protected ApiConfiguration ApiConfiguration
                         var builder = this.ConfigureApi(new ApiBuilder());
```

Comment:
Then let's leave it as now and make decision on ApiBuilder later. At least for now it helps reduce usage of CLSCompliant.